src/config/i18n/es/messages.json

```diff
@@ -1,26 +1,26 @@
 {
-  "auth.signin.description": "",
-  "auth.signin.go": "",
-  "auth.signin.title": "",
-  "auth.signup.email": "",
-  "auth.signup.password": "",
-  "auth.signup.title": "",
-  "common.close": "",
-  "footer.legal.cookies": "",
-  "footer.legal.privacy": "",
-  "footer.legal.terms": "",
-  "home.about.description": "",
-  "home.about.img": "",
-  "home.about.title": "",
-  "home.contact.description": "",
-  "home.contact.img": "",
-  "home.contact.title": "",
-  "home.header.cta": "",
-  "home.header.logo": "",
-  "home.mission.description": "",
-  "home.mission.img": "",
-  "home.mission.title": "",
-  "legal.notification": "",
-  "site.slogan": "",
-  "site.title": ""
-}
\ No newline at end of file
+  "auth.signin.description": "Entra en tu cuenta o crea una nueva cuenta para empezar",
+  "auth.signin.go": "¡Vamos!",
+  "auth.signin.title": "Entra en tu cuenta",
+  "auth.signup.email": "Correo electrónico",
+  "auth.signup.password": "Contraseña",
+  "auth.signup.title": "Crea una cuenta",
+  "common.close": "Cerrar",
+  "footer.legal.cookies": "Cookies",
+  "footer.legal.privacy": "Política de privacidad",
+  "footer.legal.terms": "Términos y condiciones",
+  "home.about.description": "Coding Coach es una plataforma gratuita y de código abierto que busca conectar a desarrolladores de software y mentores de todo el nundo. Está desarrollada y construida por un grupo de desarrolladores, diseñadores, ingenieros y sobre todo humanos, llenos de pasión y talento, con el objetivo de hacer del mundo de la ingenieria un mejor lugar para la colaboración. Este proyecto nació gracias al deseo de crear una plataforma que conectase mentores y aprendices de todo el mundo sin coste alguno. Coding Coach está creado por personas, y para las personas.",
+  "home.about.img": "Un hombre andando hacia un ordenador portatil",
+  "home.about.title": "Sobre Coding Coach",
+  "home.contact.description": "¡Nos gustaria saber tu opinión! Únete a nuestro <0>Slack</0> o envianos un correo electrónico a <1>{EMAIL}</1>",
+  "home.contact.img": "Una persona usando una lupa para ver los detalles de otra persona",
+  "home.contact.title": "Contacto",
+  "home.header.cta": "Quiero saber más",
+  "home.header.logo": "Este es el logo de Coding Coach",
+  "home.mission.description": "Creemos que el acceso a un mentor debe ser posible para cualquier persona sin importar su localización o condición económica o financiera. Para conseguir esto, hemos creado una plataforma gratuita y de código abierto para facilitar el acceso a las conexiones con mentores. Nuestra mision es formentar el sentido de la colaboración y la inclusividad en la industria técnica proporcionando una plataforma para ayudar con el proceso de mentorizado.",
```

Comment:
Use a comma for the first or: `localización o condición económica o financiera` -> `localización, condición económica o financiera`

Reply:
+1

Reply:
typo. `misión`